src/libprojectM/Renderer/BeatDetect.cpp

```diff
@@ -36,31 +36,32 @@
 #include <cmath>
 #include "BeatDetect.hpp"
 
-BeatDetect::BeatDetect(PCM *_pcm) {
-  int x,y; 
 
-  this->pcm=_pcm;
+BeatDetect::BeatDetect(PCM *_pcm)
+{
+    this->pcm=_pcm;
 
-  this->vol_instant=0;
-  this->vol_history=0;
+    this->vol_instant=0;
+    this->vol_history=0;
+    for (unsigned y=0;y<80;y++)
```

Comment:
what's the magic `80` number?

Reply:
That's from the previous code.  Would be better as a constant (like I did for FFT_LENGTH)